tests/kafkatest/services/console_consumer.py

```diff
@@ -167,7 +168,8 @@ def start_cmd(self, node):
         else:
             args['kafka_opts'] = self.security_config.kafka_opts
```

Comment:
Could we have `kafka_opts` return the adjusted ops for the new jvm?

Reply:
Hello, @ijuma 

Can you, please, clarify your proposal?

I looked at `kafka_opts` values in the tests and it's usage in `kafka-run-class.sh` and doesn't get it. How should I adjust it and why?
Can you, please, give me an example of some wrong value of `kafka_opts`?

Reply:
Hello @ijuma 

Can you, please, explain your idea to me?
So I can fix this PR according to your requirements.

Reply:
The function `kafka_opts` returns jvm options that don't work with certain JVM versions. I was suggesting that we could pass a node to it so that it returns the right jvm options. That way, we don't need to fix them after the fact. Does that makes sense?

Reply:
Hello @ijuma 

I assume you are writing about `security_config.py` kafka_opts function:
Is it correct?

If yes, then it seems all options are supported in java since JDK7 or even earlier:
Can you, please, clarify, what specific jvm options should be fixed?

https://docs.oracle.com/javase/7/docs/technotes/guides/security/jgss/tutorials/LoginConfigFile.html

```
    @property
    def kafka_opts(self, node):
        if self.has_sasl:
            if self.static_jaas_conf:
                return "\"-Djava.security.auth.login.config=%s -Djava.security.krb5.conf=%s\"" % (SecurityConfig.JAAS_CONF_PATH, SecurityConfig.KRB5CONF_PATH)
            else:
                return "\"-Djava.security.krb5.conf=%s\"" % SecurityConfig.KRB5CONF_PATH
        else:
            return ""
```

Reply:
Hello @ijuma 

Can you, please, explain your proposal for `kafka_opts` one more time :)
I really want to fix it.

Reply:
My bad. The unsupported options come directly from the shell scripts. You are overriding depending on the JVM version. Makes sense.

Reply:
Thanks for the reply!

Is everything OK with the PR?
Should I additionally check some other tests?